frontend/simple/js/state.js

```diff
@@ -7,14 +7,15 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 import * as db from './database'
 import * as Events from '../../../shared/events'
+import backend from '../js/backend'
 
 // babel transforms lodash imports: https://github.com/lodash/babel-plugin-lodash#faq
 // for diff between 'lodash/map' and 'lodash/fp/map'
 // see: https://github.com/lodash/lodash/wiki/FP-Guide
 import debounce from 'lodash/debounce'
 
 Vue.use(Vuex)
-
+/* global sessionStorage */
```

Comment:
What is this sessionStorage thing and where is it set? Is it possible to avoid globals?